_posts/2016-10-18-bio-formats-5.2.4.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Release of Bio-Formats 5.2.4
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 5.2.4
+---
+
+Today we are releasing Bio-Formats 5.2.4, a bug-fix release mainly addressing
+a regression that was introduced in 5.2.0.
+
+Improvements for two formats:
+
+*  OME-TIFF
+    *  fixed regression when populating plane metadata
+*  CZI
+    *  populated series metadata with the scene/position information
+
+Full details can be found in the [Bio-Formats version history](http://www.openmicroscopy.org/site/support/bio-formats5.2/about/whats-new.html)
+
+The software is available at 
+[downloads.openmicroscopy.org/bio-formats/5.2.4/](http://downloads.openmicroscopy.org/bio-formats/5.2.4).
```

Comment:
Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36